manim/mobject/three_dimensions.py

```diff
@@ -14,7 +14,10 @@
     "Torus",
 ]
 
+from typing import *
+
 import numpy as np
+from colour import Color
```

Comment:
nitpicky, but since `Color` is only used for type checking it should be under TYPE_CHECKING
```py
if TYPE_CHECKING:
    from colour import Color
```

Reply:
Oh i see, okay

Reply:
Am i doing something wrong? 🧐

Reply:
I think the correct thing is `if typing.TYPE_CHECKING:` instead of if `if TYPE_CHECKING:`

Reply:
@PhotonSpheres But i already had `from typing import *` above it 🤔

Reply:
Just the way I did it before was
```py
import typing
if typing.TYPE_CHECKING:
    from colour import Color
```
and this seems to work

Reply:
Nvm, i found it. `Color` was supposed to be in quotes. 😅